corehq/apps/users/tests/test_download.py

```diff
@@ -15,14 +15,20 @@
 from corehq.apps.users.models import CommCareUser
 from corehq.apps.users.bulk_download import parse_users
 from corehq.apps.user_importer.importer import GroupMemoizer
-from corehq.util.test_utils import flag_enabled
+from corehq.apps.accounting.models import SoftwarePlanEdition
+from corehq.apps.accounting.tests.utils import DomainSubscriptionMixin
 
 
-class TestDownloadMobileWorkers(TestCase):
+class TestDownloadMobileWorkers(TestCase, DomainSubscriptionMixin):
+    domain = 'bookshelf'
+
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.domain_obj = create_domain('bookshelf')
+        cls.domain_obj = create_domain(cls.domain)
+
+        # APP_USER_PROFILES is on ENTERPRISE and above
+        cls.setup_subscription(cls.domain, SoftwarePlanEdition.ENTERPRISE)
```

Comment:
You can also use `@patch('corehq.apps.user_importer.importer.domain_has_privilege', lambda x, y: True)` to avoid privileges altogether in tests.